app/models/hike.rb

```diff
@@ -60,7 +60,7 @@ def simplified_waypoints(array)
   end
 
   def average_rating
-    reviews.count > 0 ? (reviews.sum(:rating).to_f/reviews.count).round(0) : 0
+    reviews.count > 0 ? reviews.average(:rating) : 0
```

Comment:
the ternary isn't really necessary, by using count you're using two queries - `reviews.ave...` instead you could use an or clause:
`reviews.average(:rating) || 0`